### PR TITLE
Flutter tutorial post editorial review

### PR DESCRIPTION
**Pull request by Srushtika**
Final review ongoing...

### CODE REVIEW

**Comment by fliptopbox** [comment]
@Srushtika if this is ready please assign it to someone, and also add your own approval.

**Comment by Srushtika** [comment]
@fliptopbox looks like I can't be my own approver, so added you.

**Review by fliptopbox — APPROVED** [review]

File: content/tutorials/flutter-cryptocurrency-app.textile [hunk 1]
```diff
@@ -0,0 +1,847 @@
...
+# "tweet_ui":https://pub.dev/packages/tweet_ui <br> A ready-made widget to display different types of tweets by simply passing the relevant JSON data.
+
+
+h2(#realtime-charts). 2. Building the Realtime Cryptocurrency Charts
+
+Let's go back to the "Ably Cryptocurrency prices":https://www.ably.io/hub/ably-coindesk/crypto-pricing page on the Hub that you subscribed to in the previous steps. Each cryptocurrency has a display name and a code. Also, each currency has a unique channel in the Hub. We'll use the code to connect to the specific channel for the particular currency. The display name is what the user will see when using the app.
+
+Inside the @ably_service.dart@ file, we will start with a @const List@ variable that will store the currently available currencies on the Hub. If any new currency is added to the source, we can append it to this list and the whole app will be updated.
+
+```[flutter]
```
Comment (fliptopbox) [inline content/tutorials/flutter-cryptocurrency-app.textile:148]: please change the language to `dart` for all instances (there are about 4-5)

**Review by fliptopbox — CHANGES_REQUESTED** [review]

**Comment by tbedford** [comment]
I will do a more detailed review later. But for now will just add comments. Flutter icon appears missing:

image

**Comment by tbedford** [comment]
Formatting not quite right on directory structure diagram:

image

**Comment by tbedford** [comment]
Code snippets not quite right here:

image

**Comment by Srushtika** [comment]
Thanks for the comments @tbedford, I hadn't checked the preview on the website yet. I'll push some fixes. @fliptopbox as for the Flutter icon on the tutorials page, do you want me to raise a ticket for this?

**Comment by fliptopbox** [comment]
@Srushtika Nah, don't create a ticket, I will grab the SVG asset from this ...
https://commons.wikimedia.org/wiki/File:Dart-logo-wordmark.svg
and make a greyscale version.

**Comment by fliptopbox** [comment]
@Srushtika @tbedford the icons belong to the website assets, here is the PR that adds `ai-dart` and `ai-flutter` SVG icons that are used in the tutorial filter widget.

NOTE!!!!! you will need to update the languge list, so that it has the icon reference

**Review by tbedford — APPROVED** [review]
I've not run through the tutorial step by step, but have reviewed website rendering locally. The issues I raised have been fixed, and looks good.

**Review by fliptopbox — APPROVED** [review]

**Pull request merged**